tests/unit/backend/helpers/jest-express.ts

```diff
@@ -4,15 +4,18 @@ const mockRequest = <P extends Record<string, string>, B>({
   params,
   body,
   session,
+  secure,
 }: {
   params?: P
   body?: B
   session?: any
+  secure?: boolean
 } = {}) => {
   return {
     body: body ?? {},
     params: params ?? {},
     session: session ?? {},
+    secure: secure ?? {},
```

Comment:
the default value for secure shouldn't be an object right? this should be something like `secure ?? true`

Reply:
edited